recipe-server/client/actions/tests/test_preference-experiment.js

```diff
@@ -0,0 +1,136 @@
+import PreferenceExperimentAction from '../preference-experiment/';
+import { recipeFactory } from '../../tests/utils.js';
+
+function preferenceExperimentFactory({ ...args }) {
```

Comment:
What does `{ ...args }` in a parameter list do? How is this different than `function preferenceExperimentFactory(args) {`?